holoviews/plotting/mpl/element.py

```diff
@@ -105,7 +107,8 @@ class ElementPlot(GenericElementPlot, MPLPlot):
     # Element Plots should declare the valid style options for matplotlib call
     style_opts = []
 
-    _suppressed = [Table, NdElement, Collator, Columns, ItemTable]
+    # Whether plot has axes, disables setting axis limits, labels and ticks
+    _axes = True
```

Comment:
I would consider renaming this attribute ``_has_axes`` to indicate it is a boolean predicate.